CMSgov/design-system
# [WNMGDS-1355][WNMGDS-1551] Translate a11y labels for Alert and Badge and add new i18n module

## Summary

https://jira.cms.gov/browse/WNMGDS-1355
https://jira.cms.gov/browse/WNMGDS-1551

### Added

- Added an internationalization module in core that replaces `i18next`, and `@cmsgov/ds-healthcare-gov` uses this core module now

### Fixed

- Accessibility text in `Alert` and `Badge` now support Spanish

## How to test

The following core components were affected by this change:
- Alert
- Badge
- UsaBanner
- MonthPicker

The following healthcare components were affected:
- Footer
- Header

For each of these components, if they previously supported some kind of language/locale prop, make sure that old prop still works and overrides the globally-set language.
